m4_sessions/1705-inning/group_five.md

```diff
@@ -16,3 +14,45 @@ To make students realize the psychological and sociological impacts of social me
 * Jon W.
 * Mickey Fen
 * Spencer Carter
+
+## SESSION OUTCOMES
+
+* Realization of implications of social media on yourself and others
+
+* Making people think about their social presence (+/-)
+
+* Is Social Media a direct representation of who you are
+
+## Prompt questions
+
+* Government and Social Media - thoughts/ideas
+
+* What does that mean as a developer and our social responsibility 
+
+* How do you use it
+
+* What do you think of double posting? 
+
+* What do you think about people who don't use it? 
+
+## GROUPING
+  
+  * Open floor discussion
+
+## LOGISTICS
+
+| Set Up | Materials Needed | PPT Link (if PPT)|
+| ------ | ---------------- | ---------------- |
+| Room setup in a horseshoe. TVs with speakers | students need: notebooks and something to write with | netflix.com |
+
+## SESSION OUTLINE
+
+| Timing | Facilitator Talking Points | Participant Actions  |
+5 min intro / warmup - Jon
+30 min video - TV
```

Comment:
Might be good to cut the video to 20 minutes so that the discussion can go longer.  You can then send the video out to people after to keep the discussion going.  Maybe even start a follow up slack convo?